packages/electrode-archetype-opt-mocha/index.js

```diff
@@ -0,0 +1 @@
+// empty
```

Comment:
what is the need for an empty file?

Reply:
was a placeholder and now it exports an empty object.
just to make sure the import works properly to setup mocha.